### ISSUE TITLE
React module casing

### ISSUE THREAD

**Issue by TrevorJamesH** [AndrewRedican/react-json-editor-ajrm#17]
1. What version of RJEA are you using (react-json-editor-ajrm version)?
2.4.4
2. What operating system and processor architecture are you using?
OSX
3. What did you do?
Run build in CRA
4. What did you expect to see?
No errors
5. What did you see instead?

./node_modules/React/index.js
There are multiple modules with names that only differ in casing.
This can lead to unexpected behavior when compiling on a filesystem with other case-semantic.
Use equal casing. Compare these module identifiers:
* /Users/.../client/node_modules/thread-loader/dist/cjs.js!/Users.../client/node_modules/babel-loader/lib/index.js??ref--2-oneOf-2-1!/Users/.../client/node_modules/React/index.js
    Used by 1 module(s), i. e.
    /Users/.../client/node_modules/thread-loader/dist/cjs.js!/Users/.../client/node_modules/babel-loader/lib/index.js??ref--2-oneOf-2-1!/Users/.../client/node_modules/react-json-editor-ajrm/dist/index.js
* /Users/.../client/node_modules/thread-loader/dist/cjs.js!/Users.../client/node_modules/babel-loader/lib/index.js??ref--2-oneOf-2-1!/Users/.../client/node_modules/react/index.js
    Used by 183 module(s), i. e.

It's just a warning locally, but causes the build to fail on circleCI. It seems that in your minifying/uglifying process, the case of the react module is getting capitalized.

**Comment by AndrewRedican** [comment]
Hey @TrevorJamesH,

Can you paste a code snippet of how your import or require statement looks?

**Comment by TrevorJamesH** [comment]
Just the basic: import JSONInput from 'react-json-editor-ajrm' within a react component.

It's worth noting I'm using the 'next' version of create react app, using react-scripts 2.0.0-next. It could be something with the new version of webpack, but it's identifying this module as the only one using /node_modules/React

**Comment by AndrewRedican** [comment]
Hey @TrevorJamesH.

I am still looking into this. I don't have hands on experience on ReactCreateApp. I esitmate I'll be able to fix this within a week's time.

**Comment by AndrewRedican** [comment]
I was able to replicate issue using the bundled distribution version of the component.

Issue boils down to specifically an issue with Capitalization.

The ES6 import statement
```
import React fom 'react'
```

is not parsed correctly by the webpack or babel configuration, which results in an issue which really boils down to a single capital letter that is causing the warnings.


_!function(e,t){"object"==typeof exports&&"object"==typeof module?module.exports=t(require("react")):"function"==typeof define&&define.amd?define("react-json-editor-ajrm",["`R`eact"],t):"object"==typeof exports?exports["react-json-editor-ajrm"]=t(require("react")):e["react-json-editor-ajrm"]=t(e.React)}("undefined"!=typeof self?self:this,function(e)_

**Comment by AndrewRedican** [comment]
Removing these configurations from webpack solved the issue. A fix will be provided in the next release of this component.

```
    externals: {
        react: {
            commonjs  : 'react',
            commonjs2 : 'react',
            //amd       : 'React',
            //root      : 'React'
        },
        'react-dom': {
            commonjs  : 'react-dom',
            commonjs2 : 'react-dom',
            //amd       : 'ReactDOM',
            //root      : 'ReactDOM'
        }
    }
```

**Comment by AndrewRedican** [comment]
@TrevorJamesH, This has been fixed as of version `react-json-editor-ajrm@2.5.0`
